frontend/src/types/api/index.ts

```diff
@@ -7,9 +7,10 @@ export interface ErrorResponse {
 	message: null;
 }
 
-export interface SuccessResponse<T> {
+export interface SuccessResponse<T, P = unknown> {
 	statusCode: SuccessStatusCode;
 	message: string;
 	payload: T;
 	error: null;
+	params?: P;
```

Comment:
why we are making change this one ?

Reply:
you can make use of hook for same and utilise options

Reply:
it just returns our input params (body), why do we have to create a hook for that?